Update README.md - Add cmake command to fix import error in python (Windows 10)

pavitrakumar78 opened the pull request:
Added an extra command in the 'Impatient Method' for Windows to fix import error in python. 

Fixes issue #1839  

### Brief summary of changes
Added extra cmake command (to Impatient Method - Windows) to fix python import error.

### Testing I've completed
Tested on Windows 10, Python 3.5
`import opensim` works without any errors.

### Looking for feedback on...
None

### CHANGELOG.md (choose one)

-added extra cmake command with `--target` parameter

The Doxygen for this PR can be viewed at http://myosin.sourceforge.net/?C=N;O=D; click the folder whose name is this PR's number.

chrisdembia commented:
Thanks for submitting this. Could you put this after the `ctest` line? You should only install if the tests pass.

Also, could you add a note after the instructions saying "You may need to set your system's PATH environment variable to include the full path to the `opensim_install\bin` folder."

Also, could you either add similar lines to the other "for the impatient" instructions," or create GitHub Issues to track adding these in the future?

pavitrakumar78 commented:
> Thanks for submitting this. Could you put this after the ctest line? You should only install if the tests pass.
> 
> Also, could you add a note after the instructions saying "You may need to set your system's PATH environment variable to include the full path to the opensim_install\bin folder."

Done! Added instructions and extra cmake command for both Windows and Ubuntu Impatient installations. 



> Also, could you either add similar lines to the other "for the impatient" instructions," or create GitHub Issues to track adding these in the future?

Not sure how I should do this for Mac OSX. For Ubuntu, I added `make -j8 install` command after `ctest -j8`.

chrisdembia commented:
This is fantastic @pavitrakumar78. Thanks a bunch. 

Also, I'm happy the python package is now working for you.
